baker/redirects.test.ts

```diff
@@ -0,0 +1,147 @@
+import { Url } from "../clientUtils/urls/Url.js"
+import { strToQueryParams } from "../clientUtils/urls/UrlUtils.js"
+import { formatUrls } from "../site/formatting.js"
+import * as redirects from "./redirects.js"
+import { resolveInternalRedirect } from "./redirects.js"
+
+jest.mock("../settings/clientSettings.js", () => ({
+    WORDPRESS_URL: "http://localhost:8080",
+    BAKED_BASE_URL: "http://localhost:3030",
+}))
+
+const mockRedirectMap = (redirectsArr: Array<Array<string>>) => {
+    const redirectsMap: Map<string, string> = new Map()
+
+    redirectsArr.forEach((redirect) => {
+        redirectsMap.set(redirect[0], redirect[1])
+    })
+    return redirectsMap
+}
```

Comment:
Can simply use the [Map constructor](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/Map) here.

```suggestion
const mockRedirectMap = (redirectsArr: [string, string][]) => {
    return new Map(redirectsArr)
}
```

Reply:
indeed, thanks!